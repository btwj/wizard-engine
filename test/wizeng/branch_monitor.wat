(module
  (func (export "main")
    (call $start)
  )
  (func $start (export "_start")
    (local $cnt i32)
    (local.set $cnt (i32.const 10111))
    (loop $l
        (local.set $cnt (i32.sub (local.get $cnt) (i32.const 1)))
	(br_if $l (local.get $cnt))
    )
  )
)
