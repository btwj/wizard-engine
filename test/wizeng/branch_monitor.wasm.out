func "start":
  +18 br_if: [4, 1]
func "foo":
  +22 br_table: [4, 4, 4, 8]
  +41 br_if: [19, 1]
